index.js

```diff
@@ -29,11 +29,81 @@ function enterState(newState) {
 start();
 
 async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
+  const welcomeMessage = `109 Ontario Street
+You are standing in a haunting wooded area at dusk. You hear crows cawing and fluttering 
+their wings above. To the left there is a lantern. To the right is a bench. Behind you is a
+church.' 
```

Comment:
This single quote isn't closing out the string, as the string was started with a backtick.